CHANGELOG.md

```diff
@@ -123,6 +123,8 @@ New modules
 
 * `Data.Sign.Show` to show a sign
 
+* `AlgebraPropreties.Semigroup.Reasoning` adding reasoning combinators for semigroups
+
```

Comment:
I'm also not sure whether we should refer to these as `reasoning combinators`. The `Reasoning` terminology has quite a specific use in the library for syntax that chains together nicely. I'm not sure these lemmas qualify?

Reply:
We won't need `CHANGELOG` text if it moves to `Additions to...`, but the opening comment block should be rephrased...